firmware/Core/Inc/telecommands/telecommand_parser.h

```diff
@@ -10,6 +10,10 @@
 
 #define TCMD_PREFIX_STR "CTS1+"
 #define TCMD_PREFIX_STR_LEN 5
+#define TCMD_HEX_ARRAY_LEN 32
```

Comment:
Please change to `static const uint8_t`. Please rename to something like `TCMD_SHA256_LENGTH_BYTES`